common/config_parser.hpp

```diff
@@ -87,27 +90,36 @@ class MenderConfigFromFile {
 	/** Connectivity connection handling and transfer parameters */
 	ClientConnectivity connectivity;
 
-	/** Rootfs device paths */
-	string rootfs_part_A;
-	string rootfs_part_B;
+	/** Rootfs device paths. These are not parsed by the client anymore, since rootfs updates
+		are now handled by an update module. But for historical reasons, they still share config
+		files, so these options can still be in the file. */
+	// string rootfs_part_A;
+	// string rootfs_part_B;
 
-	/** Command to set active partition */
-	string boot_utilities_set_active_part;
+	/** Command to set active partition. These are not parsed by the client anymore, since
+		rootfs updates are now handled by an update module, which doesn't care about these
+		options. */
+	// string boot_utilities_set_active_part;
 
-	/** Command to get the partition which will boot next */
-	string boot_utilities_get_next_active_part;
+	/** Command to get the partition which will boot next. These are not parsed by the client
+		anymore, since rootfs updates are now handled by an update module, which doesn't care
+		about these options. */
+	// string boot_utilities_get_next_active_part;
 
 	/** Path to the device type file */
 	string device_type_file;
 
-	/** DBus configuration */
-	bool dbus_enabled = false;
+	/** DBus configuration. This option was removed in Mender v4.0.0, where we use two daemons
+		and are totally reliant on DBus to communicate between them. */
+	// bool dbus_enabled = true;
 
-	/** Expiration timeout for the control map */
-	int update_control_map_expiration_time_seconds = 0;
+	/** Expiration timeout for the control map. The Update Control feature has been removed from
+		the C++ client. */
+	// int update_control_map_expiration_time_seconds = 0;
 
-	/** Expiration timeout for the control map when just booted */
-	int update_control_map_boot_expiration_time_seconds = 600; // 10 min
+	/** Expiration timeout for the control map when just booted. The Update Control feature has
+		been removed from the C++ client. */
+	// int update_control_map_boot_expiration_time_seconds = 600; // 10 min
 
 	/** Poll interval for checking for new updates */
 	int update_poll_interval_seconds = 1800;
```

Comment:
Few lines below, we have:
```
/** Poll interval for checking for update (check-update) */
int state_script_retry_interval_seconds = 60;
```

which looks wrong to me.

I am working on something related locally so I just noticed.